app/src/examples/AnimatedStyleUpdateExample.tsx

```diff
@@ -11,13 +10,18 @@ export default function AnimatedStyleUpdateExample(): React.ReactElement {
   const randomWidth = useSharedValue(10);
 
   const config = {
-    duration: 500,
-    easing: Easing.bezierFn(0.5, 0.01, 0, 1),
+    duration: 3000,
+    velocity: 5,
```

Comment:
Setting some const velocity for start may be a good way to test things but this won't serve as a good example. We should revert changes here and add a better example that'd illustrate the case for spring with velocity (e.g. something similar to what we do in "Darg and snap" example